cpp/include/cudf/lists/contains.hpp

```diff
@@ -163,8 +164,8 @@ std::unique_ptr<column> index_of(
  * @return std::unique_ptr<column> INT32 column of `n` rows with the location of the `search_key`
  *
  * @throw cudf::logic_error If `search_keys` does not match `lists` in its number of rows
- * @throw cudf::logic_error If `search_keys` type does not match the element type in `lists`
  * @throw cudf::logic_error If `lists` or `search_keys` contains nested elements (LIST, STRUCT)
+ * @throw cudf::data_type_error If `search_keys` type does not match the element type in `lists`
```

Comment:
Please remove line 167. It was left from previous changes, no longer relevant (`index_of` now supports nested types).

Reply:
Done